tests/journals/test_runner.ps1

```diff
@@ -0,0 +1,47 @@
+$audacityExecutable=$args[0]
+$journalFile=$args[2]
+
+Write-Host "Audacity Executable: $audacityExecutable"
+Write-Host "Journal File: $journalFile"
+
+if (-not(Test-Path -Path $journalFile -PathType Leaf)) {
+    Write-Host "Journal file does not exist"
+    exit 1
+}
+
+$process = [Diagnostics.Process]::Start("$audacityExecutable", "--journal $journalFile")
+
+$completedInTime = $process.WaitForExit(180000)
```

Comment:
What is the time unit for this number?

Should this timeout be given as an argument and made to agree with whatever timeout CTest is using?